x/ethbridge/querier/querier.go

```diff
@@ -0,0 +1,77 @@
+package querier
```

Comment:
this should go into a keeper package

Reply:
added to https://github.com/cosmos/peggy/issues/52